go/client/logclient.go

```diff
@@ -226,51 +252,43 @@ func (c *LogClient) addChainWithRetry(ctx context.Context, ctype ct.LogEntryType
 	for _, link := range chain {
 		req.Chain = append(req.Chain, link)
 	}
-	httpStatus := "Unknown"
-	// Retry after 1s, 2s, 4s, 8s, 16s, 32s, 64s, 128s, 128s, ....
-	maxInterval := 128.0
+	// Retry after 1s, 2s, 4s, 8s, 16s, 32s, 64s, 128s, 128s
```

Comment:
Does the change in comment indicate a change in behaviour?  

(I think we're working on the assumption that the governing context should be responsible for making sure that this operation eventually completes, so the perpetual retry was intended.)